Libplanet/Net/Swarm.cs

```diff
@@ -1356,8 +1390,8 @@ private void TransferTxs<T>(BlockChain<T> blockChain, GetTxs getTxs)
 
             IAsyncEnumerable<Transaction<T>> fetched = GetTxsAsync<T>(
                 peer, unknownTxIds, cancellationToken);
-            var toStage = new HashSet<Transaction<T>>(
-                await fetched.ToListAsync(cancellationToken));
+            List<Transaction<T>> txs = await fetched.ToListAsync(cancellationToken);
+            var toStage = txs.ToDictionary(tx => tx, _ => true);
```

Comment:
It seems to re-broadcast all transactions from other peer. is it intended?

Reply:
This is to stage transactions from other peers. Did you talk about [this](https://github.com/planetarium/libplanet/pull/297/files#diff-9501da6c2e844a81d5e3ad3d3d5456ccR996)? I intend to broadcast all transactions again from others, do you think it is inappropriate?

Reply:
> Did you talk about this?

Yes. if we stage in this method, we also resend them through mentioned code.

> I intend to broadcast all transactions again from others, do you think it is inappropriate?

No, it seems to be appropriate. but it would be better if related test cases exist.